packages/tools/client-debugger/client-debugger-view/src/components/FluidHandleView.tsx

```diff
@@ -83,14 +85,20 @@ export function FluidHandleView(props: FluidHandleViewProps): React.ReactElement
 	}
 
 	return (
-		<Accordion
-			header={
-				<div>{`${visualTree.metadata && visualTree.metadata} : ${
-					visualTree.nodeKind
-				}`}</div>
-			}
-		>
-			<TreeDataView containerId={containerId} node={visualTree} />;
-		</Accordion>
+		<Tree>
```

Comment:
Can we unify the common aspects of this tree view into a helper component? Since we're laying out each of our view types in a similar fashion, I'm guessing we can reduce a lot of boilerplate in each of the components by extracting this into something that takes in specific pieces of data to display (label, typeMetadata, metadata, children, etc.)

Reply:
It will also let us encapsulate our usages of the unstable Tree APIs to a single location.

Reply:
Created `RenderSummaryTree` and `RenderLabel` for visualizing components.